### PR TITLE
Refactors Atmos Air Current Pushing

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Refactors air current pushing to what TG has.

Objects and mobs will no longer be thrown about endlessly, but they still will be pushed around.

The way this works is, depending on the turfs around you and how many things you have to "hold onto" factors into how much you get pushed aorund; less things to hold onto? Far more pushing.

Object throwing is going to cause issues with a fully destructable environment, due to how objects thrown *do* cause damage.

This also ensures that scrubbers and vents don't pump out/suck out tiny amounts of air, making them more useful again.

This also removes the emergency mode that was enabled, by default, on air alarms---in my testing of the supermatter, a "hell LINDA" project, I noted that scrubbers behave very very strangely, due to constantly applying their own mode and behaving automatically. Not only did this take away some aspects of atmos techs jobs, but it made going from normal scrubbing to wideband a pain (and prone to resetting); we really need scrubbers to behave properly if we're going to refactor LINDA, the supermatter, or what have you.

This also removes a lot of the magboot spam that was caused by LIDNA throwing things; virtually everyone and their brother had access to a plethora of magboots because of just how frustrating and devastating throwing things was; this has been scaled back to the CE, Atmostech, and engineers starting with magboots, but no one else; there will be 2 pairs in EVA storage, but that's it.

On that note, nuke ops won't start with magboots either; it'll have to be something they'll have to invest in---they can either choose to go with regular syndie magboots, or they can invest in the more expensive chameleon shoes (which also protect against currents).

Likewise, wizards also no longer spawn with magboots if they purchase a hardsuit.

This also will fix a lot of issues with simple mobs being able to be cheesed/exploited by getting them to get shoved into currents then they get shredded by glass....also means that Space Carp will be a terrifying threat to the station, once again  as they'll no longer instakill themselves when breaking a window.

Lastly this re-buffs the flamethrower back to its original status; it should no longer be a pitiful weapon.

This does not touch the speed at which LINDA runs; it's still going to run every half a second and still going to push you around a lot; if you get slowed and are in a high pressure area, you're going to have difficulty getting out, but it shouldn't be incredibly frustrating.

fixes: https://github.com/ParadiseSS13/Paradise/issues/12096
fixes: https://github.com/ParadiseSS13/Paradise/issues/8569

CL coming later; want to test this out thoroughly before writing it.

:cl: Fox McCloud
del: Nuke ops can no longer advanced syndie magboots
fix: Fixes an inconsistency with chameleon kit pricing--it was only 2 TC, making the no slip syndicate shoes worthless; price is now 4 TC
tweak: operatives pay 2 TC more for the no slip syndicate shoes and the chameleon kit
del: Wizard hardsuits no longer come with magboots
tweak: Syndie magboots are nuke ops only
tweak: Can purchase no slip shoes from the uplink instead of a full chameleon kit
tweak: Operatives can purchase no slip syndicate shoes
tweak: No slip shoes can protect against atmos pushing
del: removed most round-start magboots with the exception of EVA, atmostechs, CE, and engineering hardsuit dispensers
fix: Fixes carp not being a threat to the station due to killing themselves
fix: Fixes being able to exploit simple mob AI so it kills itself in an atmos death blender
tweak: vent and scrubbers work far more efficiently and quickly now
fix: Fixes wideband on scrubbers constantly resetting itself
tweak: Portable scrubbers far more effective
tweak: Flamethrowers far more effective
tweak: Nanofrost twice as expensive
tweak: regular syndicate borgs don't have magpulse by default
tweak: Atmos pushing completely refactored; it'll still push you frequently and push items around frequently, but it won't throw you. Depending on how many dense objects or turfs are around you to "grab onto" will determine how effective atmos is at pushing you around. Does not alter the speed at which LINDA runs.
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
> depending on the turfs around you and how many things you have to "hold onto" factors into how much you get pushed aorund

Is this on all intents? I would accept this functionality if you had on "resist" intent but if it's always on then you're always expecting pressure to blow you away. That's immersion breaking. You need to be actively resisting the push in pressure

**Comment by KasparoVy** [comment]
Does being armless or having less than normal grippy appendages cause you to get blown around more?

**Comment by variableundefined** [comment]
Death to death blender

**Comment by Couls** [comment]
> Does being armless or having less than normal grippy appendages cause you to get blown around more?

from looking at the code it does seem to check if you actually have hands before giving you the stability bonus.

**Comment by farie82** [comment]
Seeing you removed the cham pack for nukies. What options do they have now to go stealth? And why is this removal required to make atmos better?

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 1]
```diff
@@ -272,7 +272,6 @@
 	new /obj/item/sensor_device(src)
 	new /obj/item/key/ambulance(src)
 	new /obj/item/pinpointer/crew(src)
-	new /obj/item/clothing/shoes/magboots(src)
```
Comment (TDSSS) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: I kind of feel like paramedics should keep their magboots. Saving people from an active breach is arguably even more important/more their thing than engineers.

**Pull request merged**